docs/configuration/config-file-reference.md

```diff
@@ -1028,315 +1028,319 @@ The `table_manager_config` configures the Cortex table-manager.
 # CLI flag: -table-manager.retention-period
 [retention_period: <duration> | default = 0s]
 
-# How frequently to poll DynamoDB to learn our capacity.
-# CLI flag: -dynamodb.poll-interval
-[dynamodb_poll_interval: <duration> | default = 2m0s]
+# How frequently to poll backend to learn our capacity.
+# CLI flag: -table-manager.poll-interval
+[poll_interval: <duration> | default = 2m0s]
 
-# DynamoDB periodic tables grace period (duration which table will be
-# created/deleted before/after it's needed).
-# CLI flag: -dynamodb.periodic-table.grace-period
+# Periodic tables grace period (duration which table will be created/deleted
+# before/after it's needed).
+# CLI flag: -table-manager.periodic-table.grace-period
 [creation_grace_period: <duration> | default = 10m0s]
 
 index_tables_provisioning:
   # Enables on demand throughput provisioning for the storage provider (if
-  # supported). Applies only to tables which are not autoscaled
-  # CLI flag: -dynamodb.periodic-table.enable-ondemand-throughput-mode
+  # supported). Applies only to tables which are not autoscaled. Supported by
+  # DynamoDB
+  # CLI flag: -table-manager.index-table.enable-ondemand-throughput-mode
   [provisioned_throughput_on_demand_mode: <boolean> | default = false]
```

Comment:
I personally find the YAML config name quite confusing, compared to the CLI flag. Have you considered rename `provisioned_throughput_on_demand_mode` to `enable_ondemand_throughput_mode`?

_Same comment applies to `inactive_throughput_on_demand_mode` and same settings to `chunk_tables_provisioning` (to keep it consistent)_